### PR TITLE
New package: jcal-0.4.1

### PR DESCRIPTION
**Pull request by LinArcX**
Jalali Calendar Library:
https://github.com/fzerorubigd/jcal

### CODE REVIEW

File: srcpkgs/jcal/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+pkgname=jcal
+version=0.4.1
+revision=1
+build_wrksrc="sources"
+build_style=gnu-configure
+hostmakedepends="automake libtool"
+makedepends="readline-devel"
+short_desc="Jalali Calendar Library"
+maintainer="linarcx <linarcx@gmail.com>"
+license="GPL-3.0"
```
Comment (Hoshpak) [inline srcpkgs/jcal/template]: this should be `GPL-3.0-or-later`

Comment (LinArcX) [reply srcpkgs/jcal/template]: How you find that?

Comment (Hoshpak) [reply srcpkgs/jcal/template]: I looked at the source files, e.g. jcal.c states:

```
 * jcal is free software: you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
```

`GPL-3.0-or-later` is the correct SPDX identifier for that case.

Comment (LinArcX) [reply srcpkgs/jcal/template]: Thank you for clarification.
Done!

**Pull request merged**